code/game/area/magmoor_digsite.dm

```diff
@@ -0,0 +1,374 @@
+// Magmoor Digsite IV - Hathkar
+
+//Base Area
+
+/area/magmoor
+	name = "Lava"
+	icon_state = "lava"
+	var/outside = TRUE/FALSE
```

Comment:
```suggestion
	var/outside = TRUE/FALSE
```

Sorry this was my bad you were meant to pick one or the other not both.

Reply:
Fixed.